packages/connect/src/index.ts

```diff
@@ -34,30 +40,23 @@ class HoprConnect implements Transport<HoprConnectDialOptions, HoprConnectListen
 
   public discovery: Discovery
 
-  private publicNodes?: PublicNodesEmitter
```

Comment:
bundle necessary config in `options` and *optional* config in `testingOptions` and then pass these objects throug entire `hopr-connect`